platform/src/main/java/org/stellar/anchor/platform/rpc/RpcMethodHandler.java

```diff
@@ -163,6 +164,13 @@ private void updateTransaction(JdbcSepTransaction txn, T request) throws AnchorE
     boolean shouldClearMessageStatus =
         !isErrorStatus(nextStatus) && isErrorStatus(SepTransactionStatus.from(txn.getStatus()));
 
+    txn.setUserActionRequiredBy(null);
+    if (request instanceof SupportsUserActionRequiredBy
+        && ((SupportsUserActionRequiredBy) request).getUserActionRequiredBy() != null) {
+      txn.setUserActionRequiredBy(
```

Comment:
Should we verify that the deadline is not in the past?

Reply:
Good point